packages/website-frontend/src/app/repository/repository.component.html

```diff
@@ -1 +1,48 @@
-<div>{{ value.slug }}</div>
\ No newline at end of file
+<div class="row">
```

Comment:
There is a lot of modal dialog logic in this template. Might it be possible to create a separate component for it? We can use [multi content projection](https://codecraft.tv/courses/angular/components/content-projection/#_multi_content_projection)

Another issue is that the modal dialog is available in the DOM for each repository in the list. Moving this dialog to the `RepositoriesComponent` would prevent this as it would only be once in the list. We could add a `toggled` event to the `RepositoryComponent` and subscribe to that in the `RepositoriesComponent`.

Reply:
I agree, I'll change it. I hope I have enough time though. Deadline is nearing quickly.

Reply:
there is no hard deadline :) But it would be cool to have it.